lib/assets/schemas/collections/umm-c-json-schema.json

```diff
@@ -300,6 +300,12 @@
       "description": "This sub-element if true, describes to end users and machines that this collection's data is free of charge and open for any use the user sees fit.",
       "type": "boolean"
     },
+    "EULAIdentifierType": {
```

Comment:
EULAIdentifiersType or EULAIdentifierType?

Reply:
It is EULAIdentifierType and not with an 's'. I got this definition from Unified Metadata Model repo (https://git.earthdata.nasa.gov/projects/EMFD/repos/unified-metadata-model/browse/collection/v1.17.2/umm-c-json-schema.json)